vmdb/tools/fix_auth/auth_model.rb

```diff
@@ -25,15 +25,19 @@ def selection_criteria(include_v2 = false)
         end.join(" OR ")
       end
 
+      def hardcode(_old_value, new_value)
+        MiqPassword.encrypt(new_value)
+      end
+
       def recrypt(old_value, options = {})
         if options[:hardcode]
-          MiqPassword.encrypt(options[:hardcode])
+          hardcode(old_value, options[:hardcode])
         else
           MiqPassword.new.recrypt(old_value)
         end
       rescue
         if options[:invalid]
-          MiqPassword.encrypt(options[:invalid])
+          hardcode(old_value, options[:invalid])
```

Comment:
I don't understand this, it looks like the method was extracted but with the same behavior. Can you help me understand what this changes?